marathon_lb.py

```diff
@@ -1407,6 +1407,24 @@ def get_apps(marathon):
                     if not alive:
                         continue
 
+            if marathon.health_check() and 'readinessChecks' in app and \
+               len(app['readinessChecks']) > 0:
+                ready = False
+                if 'readinessCheckResults' not in app:
+                    # use previously cached result, if it exists
+                    if not readinessCheckResultCache.get(task['id'], False):
+                        continue
+                else:
+                    if len(app['readinessCheckResults']) == 0:
```

Comment:
After a task is ready it seems that the `readinessCheckResults` array is always empty. Please confirm this is the correct behavior.